### PR TITLE
1.0.0

### PR DESCRIPTION
**Pull request by rreinold**
overhaul

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,16 +3,16 @@
         "email": "tboon@clearblade.com"
     },
     "clouds": [],
-    "description": "Data Table Portal UI Example based upon the DataTable.net framework",
+    "description": "Using DataTable.net framework to visualize, search, and filter ClearBlade Collections",
     "domains": [],
     "gateways": [],
     "image_url": "https://i.imgur.com/itDaijo.png",
     "keywords": [],
```
Comment (yashjain28) [inline package.json]: You might want to add keywords

File: system.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
     "auth": true,
     "description": "Data Table Portal UI Example based upon the DataTable.net framework",
-    "messaging_url": "messaging.clearblade.com:1883",
-    "name": "Data Table Example Portal",
-    "platform_url": "https://platform.clearblade.com",
-    "system_key": "eca4ffa40bf6e3a9f4ad83e1bc5d",
-    "system_secret": "ECA4FFA40BC8A999A09CB98DB5C001"
+    "messaging_url": "staging.clearblade.com:1883",
+    "name": "dev-data-table",
```
Comment (yashjain28) [inline system.json]: The name shouldn't be dev- right?
And remove systemKey and secret as you make these changes

**Review by yashjain28 — CHANGES_REQUESTED** [review]
small changes. No verification needed, once changes are made

File: package.json (hunk 1 above)
Comment (rreinold) [reply package.json]: 189bc07

File: system.json (hunk 2 above)
Comment (rreinold) [reply system.json]: 9a4270e

Comment (rreinold) [reply system.json]: system key and secret, not atm

**Pull request merged**